lib/engine/tile.rb

```diff
@@ -63,10 +63,14 @@ def self.from_code(name, color, code, **opts)
     def self.part(type, params, cache)
       case type
       when 'path'
+        num_lanes = 1
```

Comment:
i don't think this is necessary, you could just check if it's nil, because default is 1

Reply:
I needed an integer to iterate over later.

Reply:
you could just do

(num_lanes || 1).map do |...|

Reply:
OK. My Ruby newbie is showing again.